src/bin.ts

```diff
@@ -82,6 +83,49 @@ if (version) {
         'general',
         `🚀 Starting Lost Pixel in 'platform' mode`,
       );
+      log.process(
+        'info',
+        'general',
+        process.env.LOST_PIXEL_PLATFORM_DEBUG_MODE,
+      );
+      log.process('info', 'general', process.env.GITHUB_WORKFLOW);
+      log.process('info', 'general', isPlatformDebugMode());
```

Comment:
this is debug only, right?
```suggestion
```

Reply:
yups, this one will go away together with github image build